spec/factories/participant_id_changes.rb

```diff
@@ -3,7 +3,7 @@
 FactoryBot.define do
   factory :participant_id_change do
     user
-    association :from_participant, factory: :user
-    association :to_participant, factory: :user
+    from_participant_id { SecureRandom.uuid }
+    to_participant_id { SecureRandom.uuid }
```

Comment:
minor fix, i think it should be:
```ruby
to_participant_id { user.ecf_id }
```
As we're changing to current user (not all records, but if makes sense to keep this for the factory)